exercises/concept/animal-magic/.meta/exemplar.go

```diff
@@ -1,3 +1,4 @@
+//nolint:gosec // Randomness is part of an exercise where math.rand is more appropriate than crypto.rand
```

Comment:
I found the phrasing a bit hard to follow and the two comment expressions in one line seemed strange. What do you think about the following instead? (If you like it, please apply it to all the places.)

```suggestion
// In the context of the exercise, it is fine to use math.Rand instead of crypto.Rand.
// The following directive will deactivate the respective linter check.
//nolint:gosec
```